src/components/MenuItem.js

```diff
@@ -124,13 +123,14 @@ const MenuItem = props => (
                             />
                         </View>
                     )}
+                    {props.shouldShowSelectedState && <SelectCircle isChecked={props.isSelected} />}
```

Comment:
Just FYI I tried to help out a bit and moved this change to a new PR to lighten things up here.